server/logger.js

```diff
@@ -9,31 +9,15 @@ if (!fs.existsSync(logDir)) {
   fs.mkdirSync(logDir);
 }
 
-const logger = new winston.Logger({
-  transports: [
-    new winston.transports.File({
-      level: env === 'development' ? 'debug' : 'info',
-      filename: logDir.concat('/logs.log'),
-      handleExceptions: true,
-      json: true,
-      maxsize: 5242880, // 5MB
-      maxFiles: 5,
-      colorize: false
-    }),
-    new winston.transports.Console({
-      level: 'debug',
-      handleExceptions: true,
-      json: false,
-      colorize: true
-    })
-  ],
-  exitOnError: false
+const logger = new blgr();
+logger.set({
+  level: env === 'development' ? 'debug' : 'info',
+  colors: true,
+  filename: logDir.concat('/logs.log')
 });
 
-/*
-  Stream is used to allow morgan to style console output
-  in server.js
-*/
+logger.open();
```

Comment:
This is also an async operation. I wonder if this could cause some problems when enabling file logging.